platform.md

```diff
@@ -1328,6 +1340,7 @@ command = ["<command>"]
 args = ["<arguments>"]
 direct = false
 working-dir = "<working directory>"
+exec-env = ["<execution environment>"] # Optional. If not specified, applies to all execution environments. The special value "*" indicates compatibility with all execution environments. If exec-env is ["*"] or not specified, the process is eligible for all execution environments.
```

Comment:
Does this align with the following?
> Documented that it defaults to "production" when not specified

I know the default input for the phases would be `production` when not set - I guess the buildpack could produce processes for `production` , `test` or omit the value in its `launch.toml`. 

Is lifecycle going to record these launch toml values as `*` or omit? I would think `*`. I don't see a reason to omit values in this file/metadata.json